commonmark/src/main/java/org/commonmark/internal/BlockQuoteParser.java

```diff
@@ -47,13 +56,28 @@ private static boolean isMarker(ParserState state, int index) {
     public static class Factory extends AbstractBlockParserFactory {
         public BlockStart tryStart(ParserState state, MatchedBlockParser matchedBlockParser) {
             int nextNonSpace = state.getNextNonSpaceIndex();
+            String preBlockWhitespace = "";
+            
+            if(nextNonSpace > 0) {
```

Comment:
Sorry about that. It looks like my IDE didn't have an editorconfig plugin, so many of my changes had tabs instead of 4 spaces. I'll fix anything I can find in the next push.